Sources/AblyChat/Version.swift

```diff
@@ -0,0 +1,16 @@
+import Ably
+
+// Update this when you release a new version
+
+// Version information
+public let version = "0.1.0"
```

Comment:
Shouldn't it be in a config file?

Reply:
purely here for the header stuff which we want to change anyway. Have added a todo